### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by gkjohnson** [gkjohnson/three-bvh-csg#72]
Add more publicly documented classes, parameters, demos, example snippets.

**Comment by AngyDev** [comment]
Hi @gkjohnson, do you want to create a documentation like the three-mesh-bvh README.md file?

**Comment by gkjohnson** [comment]
Yeah that would be perfect - the docs that are there should already be structured a bit similarly in this project, too. Hopefully the documentation structure is self-explanatory but if feel free to ask questions!

**Comment by AngyDev** [comment]
Ok I try, let's see what I can do 😊
